tests/accounts/base.py

```diff
@@ -55,7 +55,7 @@ def create_account(self, **overrides):
             'site_id': self.site_id,
             'url': self.url,
             'user_name': self.user_name,
-            'user_id': self.user_email,
+            'user_id': '{}@yola.com'.format(uuid4().hex),
```

Comment:
Why did you change that line? Was it working correctly before?

Reply:
when I switch to pytest it starts raise exception that this user already registered

Reply:
Wonder how it was working before 🤔